### PR TITLE
Divide Vendor Program and How to Prepare App Service

### PR DESCRIPTION
**Pull request by Adam-Dupaski**

### CODE REVIEW

File: content/appstore/creating-content/as-prepare.md [hunk 1]
```diff
@@ -1,90 +1,32 @@
 ---
-title: "Sell Your App Service in the Marketplace"
+title: "Prepare Your App Service to Sell in the Marketplace"
```
Comment (KseniiaIankina) [inline content/appstore/creating-content/as-prepare.md:2]: Prepare your Commercial content to Sell in Marketplace.

Then we could have more granular view on various content types.

File: content/appstore/creating-content/as-prepare.md [hunk 2]
```diff
@@ -1,90 +1,32 @@
...
 category: "Creating Content"
-menu_order: 1
-description: "Describes how to start selling your app service in the Mendix Marketplace."
+menu_order: 2
+description: "Describes how to prepare your your app service to sell in the Mendix Marketplace."
```
Comment (KseniiaIankina) [inline content/appstore/creating-content/as-prepare.md:5]: your commercial content

File: content/appstore/creating-content/as-prepare.md [hunk 3]
```diff
@@ -1,90 +1,32 @@
...
 ---
 
-## 1 Introduction
+{{% todo %}}[**SHOULD THIS DOC EVEN BE PUBLISHED IF THIRD-PARTY PRICING MODELS & SELLING FUNCTIONALITY WILL NOT BE AVAIALBLE THIS FY?**]{{% /todo %}}
 
-Whether you are an established ISV, an ambitious startup, or a pioneering independent developer, by [sharing](/appstore/general/share-app-store-content) your [app service](/appstore/app-services/) in the Mendix Marketplace, you will create customer value by fueling the resource library for Mendix developers. The process is simple: sign up as a [Mendix Vendor](#vendor), design and build your app service, edit your listing, and submit it to the Marketplace today. In return, you will gain the following benefits:
+## 1 Introduction
 
-![](attachments/as-sell/benefits.png)
+To sell your app service in the Mendix Marketplace, you need to first sign up as a [Mendix Vendor](vendor-program). Once that process is complete, you need to prepare your app service to sell in the Marketplace.
```
Comment (KseniiaIankina) [inline content/appstore/creating-content/as-prepare.md:13]: We do not allow selling but we allow to have Lead generation for Beta suppliers.

To bring your commercial product to the Mendix Marketplace, you need to first sign up as a [Mendix Vendor](vendor-program). Once that process is complete, you need to prepare your content to sell in the Marketplace. 
Currently, this process is restricted to the Beta suppliers, so you have to be part of the Beta program.

File: content/appstore/creating-content/as-prepare.md [hunk 4]
```diff
@@ -1,90 +1,32 @@
...
-* Mendix reserves the right to reject app services that do not comply with these guidelines or the requirements described in the license agreement
-
-### 2.4 Legal & Compliance Checks
-
-The company registered with your account is the legal entity against which Mendix will be bound. For this reason, Mendix performs screening checks to ensure we are legally permitted to undertake business transactions together.
-
-In addition, before your app service can be published in the Marketplace, it must be classified for export to the countries in which it will be sold and for how it will be used. For details, see the [Export Control](#export-control) section below.
-
-For more information on what you need to do as a Mendix Vendor to submit an app service, see the sections below.
+* Be registered and approved as a [Mendix Vendor](vendor-program)
```
Comment (KseniiaIankina) [inline content/appstore/creating-content/as-prepare.md:23]: Be part of the Beta program

File: content/appstore/creating-content/vendor-program.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+Whether you are an established ISV, an ambitious startup, or a pioneering independent developer, by [sharing](/appstore/general/share-app-store-content) your content in the Mendix Marketplace, you will create customer value by fueling the resource library for Mendix developers. The process is simple: sign up as a [Mendix Vendor](https://www.mendix.com/marketplace-vendor-program/), design and build your content, edit your listing, and submit it to the Marketplace.
+
+In return, you will gain the following benefits:
+
+{{% todo %}}[**WANT TO EDIT THIS DIAGRAM TO BE GENERIC? OR REMOVE ENTIRELY?**]{{% /todo %}}
+
+![](attachments/as-sell/benefits.png)
+
+The [Mendix Vendor Program](https://www.mendix.com/marketplace-vendor-program/) includes opportunities for a variety of types of partners. We only need some key information from you before you are ready to offer your product through the Mendix Marketplace. Mendix will work with you throughout the application process to make your submission experience as simple as possible.
+
```
Comment (KseniiaIankina) [inline content/appstore/creating-content/vendor-program.md:22]: Can we add here a section explaining what is the Beta program

File: content/appstore/creating-content/as-prepare.md [hunk 6]
```diff
@@ -1,90 +1,32 @@
...
-Whether you are an established ISV, an ambitious startup, or a pioneering independent developer, by [sharing](/appstore/general/share-app-store-content) your [app service](/appstore/app-services/) in the Mendix Marketplace, you will create customer value by fueling the resource library for Mendix developers. The process is simple: sign up as a [Mendix Vendor](#vendor), design and build your app service, edit your listing, and submit it to the Marketplace today. In return, you will gain the following benefits:
+## 1 Introduction
 
-![](attachments/as-sell/benefits.png)
+To sell your app service in the Mendix Marketplace, you need to first sign up as a [Mendix Vendor](vendor-program). Once that process is complete, you need to prepare your app service to sell in the Marketplace. 
 
 **This how-to will teach you how to do the following:**
 
-* Join the Mendix Vendor Program
 * Make your app service ready to sell on the Marketplace
```
Comment (KseniiaIankina) [inline content/appstore/creating-content/as-prepare.md:17]: Make your content ready to sell on the Marketplace

File: content/appstore/creating-content/as-prepare.md [hunk 7]
```diff
@@ -1,90 +1,32 @@
...
-* Complete specific tasks before you submit, as you submit, and after you submit your app service to the Mendix Marketplace
```
Comment (KseniiaIankina) [inline content/appstore/creating-content/as-prepare.md:21]: Complete specific tasks as you submit, and after you submit your commercial content

File: content/appstore/creating-content/as-prepare.md [hunk 8]
```diff
@@ -1,90 +1,32 @@
...
-In addition, before your app service can be published in the Marketplace, it must be classified for export to the countries in which it will be sold and for how it will be used. For details, see the [Export Control](#export-control) section below.
-
-For more information on what you need to do as a Mendix Vendor to submit an app service, see the sections below.
+* Be registered and approved as a [Mendix Vendor](vendor-program)
 
 ## 3 Making Your App Service Marketplace-Ready
 
 The Marketplace provides a trustworthy experience for sharing and consuming app services.  This is achieved through proven curation and governance processes as well as compliance checks conducted at the point of consumption. To walk you through the process of submitting content to the Marketplace, the intuitive [Upload Content](/appstore/general/share-app-store-content#adding) wizard provides steps for adding and publishing your app service.
 
+{{% todo %}}[**HOWEVER, THERE ARE NO FEATURES FOR MONETIZING/SELLING CONTENT IN THAT WIZARD - HOW TO DESCRIBE MECHANISM/FLOW FOR SELLING CONTENT?**]{{% /todo %}}
```
Comment (KseniiaIankina) [inline content/appstore/creating-content/as-prepare.md:28]: As step one we offer "Lead generation" . It is not yet released - in the upcoming month we will. But i believe this whole page has to be updated

File: content/appstore/creating-content/vendor-program.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+* Your content must not include software that collects and exports customer data without the customer's knowledge and express consent
+* You must define the applicable laws, treaties, regulations for your content, including, without limitation, those related to export control, data privacy, international communications, and the transmission of technical or personal data
+* Your content must be production-ready and must include a defined support policy
+* Mendix reserves the right to reject content that does not comply with these guidelines or the requirements described in the license agreement
+
+## 4 Legal & Compliance Checks
+
+The company registered with your account is the legal entity against which Mendix will be bound. For this reason, Mendix performs screening checks to ensure we are legally permitted to undertake business transactions together.
+
+{{% todo %}}[**IS THIS PART ACCURATE? IS THIS REQUIREMENT GENERIC TO ALL COMPONENTS - STARTING WITH SOLUTIONS - SOLD VIA THE VENDOR PROGRAM, OR IS IT SPECIFIC TO APP SERVICES?**]{{% /todo %}}
```
Comment (KseniiaIankina) [inline content/appstore/creating-content/vendor-program.md:68]: We have a governance page with the list of valid reqirements. These checks are perfomed for all the supplied content. https://paper.dropbox.com/doc/Consolidated-Governance-Papers--BcBpvS05rKuyO~8y9U8JgA1cAg-hFMWH3MMXbtRDJ7DoLJ1N

File: content/appstore/creating-content/vendor-program.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+| Account Owner | The account creator’s email address. |
+| Logo | Your supplier logo. |
+| Name | Supplier name (could be a company, brand, or individual). |
+| Description | Describe your purpose, products, and services. |
+| Company Name | The legal name of the entity that will sell or offer your content. |
+| Company Address | The legal address of the entity that will sell or offer your content. |
+
+As the owner of the Mendix Vendor account, you must have permission to legally bind your company to Mendix.
+
+If you want to offer a paid service, you must also provide the following details:
```
Comment (KseniiaIankina) [inline content/appstore/creating-content/vendor-program.md:40]: I think for the time being we can delete it and if it is required as part of the ISV, i will be manually asked

File: content/appstore/creating-content/vendor-program.md [hunk 11]
```diff
@@ -0,0 +1,75 @@
+---
+title: "Mendix Vendor Program"
```
Comment (KseniiaIankina) [inline content/appstore/creating-content/vendor-program.md:2]: Let's do this split but then the "Prepare your app service" sections needs full review and rework.I will be responsible to bring content

File: content/appstore/creating-content/vendor-program.md [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+| Account Number | The account number for the legal entity or individual. |
+| Account Holder Name | The name registered with the account. |
+
+## 3 Mendix Vendor License Agreement {#vendor-license}
+
+Whether you are offering a free or paid component through the Marketplace, you need to accept our Mendix Vendor License Agreement, which is an agreement between Mendix and you as a Mendix Vendor within the Mendix Vendor Program.
+
+For ease of reference, here are some of the main points of the license agreement you need to be aware of:
+
+{{% todo %}}[**THESE GENERIC POINTS NEED VERIFICATION**]{{% /todo %}}
```
Comment (KseniiaIankina) [inline content/appstore/creating-content/vendor-program.md:55]: I will align with Legal and confirm this list. Have a call next week

File: content/appstore/creating-content/vendor-program.md (hunk 10 above)
Comment (Adam-Dupaski) [reply content/appstore/creating-content/vendor-program.md:40]: @KseniiaIankina 

Which part: 

"As the owner of the Mendix Vendor account, you must have permission to legally bind your company to Mendix."

or 

" If you want to offer a paid service, you must also provide the following details: " + list

or both?

File: content/appstore/creating-content/vendor-program.md (hunk 5 above)
Comment (Adam-Dupaski) [reply content/appstore/creating-content/vendor-program.md:22]: Added "The Mendix Vendor program is a Beta program, and is currently only applicable for content other than solutions and solution templates."

Not sure how to add "So there are ISV and Beta programs." How do we define the difference between the "Mendix Vendor Beta program" and an "ISV program"?

File: content/appstore/creating-content/vendor-program.md (hunk 9 above)
Comment (Adam-Dupaski) [reply content/appstore/creating-content/vendor-program.md:75]: Not sure which Papers from that list are finalized and which should be incorporated here. Can you please clarify?

File: content/appstore/creating-content/vendor-program.md (hunk 10 above)
Comment (KseniiaIankina) [reply content/appstore/creating-content/vendor-program.md:44]: " If you want to offer a paid service, you must also provide the following details: " + list

this one

File: content/appstore/creating-content/vendor-program.md (hunk 5 above)
Comment (KseniiaIankina) [reply content/appstore/creating-content/vendor-program.md:22]: can we modify with the following logic:
The Mendix Vendor program consists of Mendix Solution Vendor and Beta Supplier programs. The latter is currently applicable for content other than solutions and solution templates. If you are interested in particapating in the Beta program, please reach out to appservices_supplier_team@mendix.com. If you want to upload commercial solution, leave a request via the form on https://www.mendix.com/marketplace-vendor-program/solutions/

File: content/appstore/creating-content/vendor-program.md (hunk 9 above)
Comment (Adam-Dupaski) [reply content/appstore/creating-content/vendor-program.md:63]: We can also add governance content after merging this PR, as part of further editing/restructuring.

File: content/appstore/creating-content/vendor-program.md (hunk 5 above)
Comment (Adam-Dupaski) [reply content/appstore/creating-content/vendor-program.md:22]: @KseniiaIankina can this be addressed, or does this also need Legal's input?

File: content/appstore/creating-content/vendor-program.md (hunk 9 above)
Comment (KseniiaIankina) [reply content/appstore/creating-content/vendor-program.md:63]: i agree, let's do !

File: content/appstore/creating-content/vendor-program.md (hunk 5 above)
Comment (KseniiaIankina) [reply content/appstore/creating-content/vendor-program.md:22]: Legal input is not required here, this we can add

File: content/appstore/creating-content/vendor-program.md (hunk 12 above)
Comment (Adam-Dupaski) [reply content/appstore/creating-content/vendor-program.md:46]: So, last comment to resolve after your contact with Legal, and then we can publish these first changes.

File: content/appstore/creating-content/vendor-program.md [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+
+* Your content must be free from any known security vulnerabilities
+* Your content must not contain default passwords, authorization keys, or any other credentials
+* Your content must not include software that collects and exports customer data without the customer's knowledge and express consent
+* You must define the applicable laws, treaties, regulations for your content, including, without limitation, those related to export control, data privacy, international communications, and the transmission of technical or personal data
+* Your content must be production-ready and must include a defined support policy
+* Mendix reserves the right to reject content that does not comply with these guidelines or the requirements described in the license agreement
+
+## 4 Legal & Compliance Checks
```
Comment (KseniiaIankina) [inline content/appstore/creating-content/vendor-program.md:55]: The company registered with your account is the legal entity against which Mendix will be bound. For this reason, Mendix performs screening checks to ensure we are legally permitted to undertake business transactions together.

You, as the company, are responsible for ensuring that you comply with all local laws and regulations in the countries where you are conducting business.

File: content/appstore/creating-content/vendor-program.md (hunk 12 above)
Comment (KseniiaIankina) [reply content/appstore/creating-content/vendor-program.md:46]: We shall delete For ease of reference, here are some of the main points of the license agreement you need to be aware of: and everything beyond .

Comment (Adam-Dupaski) [reply content/appstore/creating-content/vendor-program.md]: Removed list, thanks.

File: content/appstore/creating-content/vendor-program.md (hunk 13 above)
Comment (Adam-Dupaski) [reply content/appstore/creating-content/vendor-program.md:44]: Replaced section with these 2 sentences.

**Pull request merged**